bluemira/geometry/compound.py

```diff
@@ -63,6 +63,11 @@ def _create(cls, obj: cadapi.apiCompound, label="") -> BluemiraCompound:
         bm_shells = [BluemiraShell._create(shell) for shell in cadapi.shells(obj)]
         bm_faces = [BluemiraFace._create(face) for face in cadapi.faces(obj)]
         bm_wires = [BluemiraWire(wire) for wire in cadapi.wires(obj)]
+        if len(bm_wires) == 0:
```

Comment:
Ok what does this enable? MAybe leave a comment about why

Reply:
Sure, in some weird edge case I found there were only edges and no wires will add